app/src/main/java/cm/aptoide/pt/download/FileDownloadTask.java

```diff
@@ -119,6 +122,14 @@ protected void paused(BaseDownloadTask baseDownloadTask, long soFarBytes, long t
         new FileDownloadTaskStatus(AppDownloadStatus.AppDownloadState.WARN, md5, null));
   }
 
+  private boolean isMd5Approved(String md5, String fileName) {
+    if (shouldConfirmMd5) {
```

Comment:
Should we inject shouldConfirmMd5 in method? doesn't change anything but we would avoid the method having do depend on class variables.